packages/front-end/components/Experiment/StartExperimentBanner.tsx

```diff
@@ -0,0 +1,264 @@
+import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
+import { FeatureInterface } from "back-end/types/feature";
+import { SDKConnectionInterface } from "back-end/types/sdk-connection";
+import { VisualChangesetInterface } from "back-end/types/visual-changeset";
+import Link from "next/link";
+import { MatchingRule } from "shared/util";
+import { MdRocketLaunch } from "react-icons/md";
+import { ReactElement } from "react";
+import {
+  FaCheckSquare,
+  FaMinusSquare,
+  FaRegSquare,
+  FaTimes,
+} from "react-icons/fa";
+import track from "@/services/track";
+import { useAuth } from "@/services/auth";
+import Button from "../Button";
+import Tooltip from "../Tooltip/Tooltip";
+
+export function StartExperimentBanner({
+  experiment,
+  linkedFeatures,
+  visualChangesets,
+  connections,
+  mutateExperiment,
+  editPhase,
+  newPhase,
+}: {
+  experiment: ExperimentInterfaceStringDates;
+  linkedFeatures: {
+    feature: FeatureInterface;
+    rules: MatchingRule[];
+  }[];
+  visualChangesets: VisualChangesetInterface[];
+  connections: SDKConnectionInterface[];
+  mutateExperiment: () => unknown | Promise<unknown>;
+  newPhase?: (() => void) | null;
+  editPhase?: ((i: number | null) => void) | null;
+}) {
+  const { apiCall } = useAuth();
+
+  if (experiment.status !== "draft") return null;
+
+  type CheckListItem = {
+    display: string | ReactElement;
+    status: "error" | "skip" | "success" | "unknown";
+    tooltip?: string | ReactElement;
+  };
+  const checklist: CheckListItem[] = [];
+
+  // At least one linked change
+  const hasLinkedChanges =
+    linkedFeatures.length > 0 || visualChangesets.length > 0;
+  checklist.push({
+    display:
+      "At least one linked change (Feature Flag or Visual Editor) added to experiment",
+    status: hasLinkedChanges ? "success" : "error",
+  });
+
+  // No unpublished feature flags
+  const hasFeatureFlagsErrors = linkedFeatures.some((f) =>
+    f.rules.some(
+      (r) => r.draft || !r.environmentEnabled || r.rule.enabled === false
```

Comment:
RE: the `r.environmentEnabled` check - Do we want to verify that every environment is enabled for every feature? For example, I may have a rule that I want to disable for specific environments and only run in prod and dev ... Not having all environments enabled for a feature results in a red x in the pre-launch checklist which can seem like something is wrong.
<img width="635" alt="image" src="https://github.com/growthbook/growthbook/assets/2374625/f33fd1ae-2135-4e02-b780-35eccfaaf31d">

Reply:
A couple options:
1. Change the check to just require at least one rule to be live per feature
2. Introduce another checkbox state with a yellow warning triangle and a tooltip, so if you have at least one active rule, but some disabled ones, it would warn you, but not block you from starting.

Reply:
1. makes most sense to me. My understanding is that it is normal to have features enabled for some environments and not for others. I'm not sure if there are any indirect repercussions that would degrade the integrity of an experiment by doing so ... Given that it's a normal workflow, 2. seems more alarming than it should be.